test/query_generator/naming/common.dart

```diff
@@ -0,0 +1,75 @@
+const schema = r'''
+enum Enum {
+  e1
+  e2
+}
+
+input Input {
+  i: String
+  e: Enum
+  s: SubInput
+}
+
+input SubInput {
+  s: String
+}
+
+type Thing {
+  id: String
+  e: Enum
+  aThing: Thing
+  bThing: Thing
+  fThing: Thing
+}
+
+type Query {
+  thing(input: Input!): Thing
```

Comment:
May be we could select some more meaningful schema definition for tests and make it similar across the tests. I understand that due to the lack of time this is a solution but it makes the further refactoring more complex. Also the schema signature variates a lot between tests which adds additional complexity